File: backend/src/main/java/com/back/back9/domain/tradeLog/dto/TradeLogRequest.java
package com.back.back9.domain.tradeLog.dto;

import org.springframework.format.annotation.DateTimeFormat;

import java.time.LocalDate;

public record TradeLogRequest(
        String type,
        Integer coinId,
        Integer siteId,
        @DateTimeFormat(iso = DateTimeFormat.ISO.DATE) LocalDate startDate,
        @DateTimeFormat(iso = DateTimeFormat.ISO.DATE) LocalDate endDate
) {}

File: backend/src/main/kotlin/com/back/back9/domain/tradeLog/dto/TradeLogRequest.kt
package com.back.back9.domain.tradeLog.dto

import org.springframework.format.annotation.DateTimeFormat
import java.time.LocalDate

// 프론트 요청 DTO
data class TradeLogRequest(
    val type: String?,
    val coinId: Int?,
    val siteId: Int?,
    @field:DateTimeFormat(iso = DateTimeFormat.ISO.DATE)
    @param:DateTimeFormat(iso = DateTimeFormat.ISO.DATE)
    val startDate: LocalDate?,
    @field:DateTimeFormat(iso = DateTimeFormat.ISO.DATE)
    @param:DateTimeFormat(iso = DateTimeFormat.ISO.DATE)
    val endDate: LocalDate?
)